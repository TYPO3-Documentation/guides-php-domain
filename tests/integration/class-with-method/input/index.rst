=================================
PHP Class with implicit namespace
=================================

..  php:class:: TYPO3\CMS\Core\Test

    Lorem Ipsum Dolor!

    ..  php:method:: setDate(int $year, int $month, int $day)

            Set the date.

    ..  php:method:: getDate(): int

            Get the date.
