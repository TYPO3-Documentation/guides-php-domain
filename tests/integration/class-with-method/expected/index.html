<!-- content start -->
    <div class="section" id="php-class-with-explicit-namespace">
            <h1>PHP Class with explicit namespace</h1>
            <dl class="php class">
    <dt class="sig sig-object php"
        id="typo3-cms-core-test">
        <em class="property"><span class="pre">class</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\</span></span>
<span class="sig-name descname"><span class="pre">Test</span></span><a class="headerlink" href="#typo3-cms-core-test" title="Permalink to this definition">¶</a>    </dt>
    <dd>

    <p>Lorem Ipsum Dolor!</p>
<dl class="php method">
    <dt class="sig sig-object php" id="typo3-cms-core-test-setdate">
<span class="sig-name descname"><span class="pre">setDate</span></span>
<span class="sig-paren">(</span>
<em class="sig-param"><span class="pre">int $year</span></em>, <em class="sig-param"><span class="pre">int $month</span></em>, <em class="sig-param"><span class="pre">int $day</span></em><span class="sig-paren">)</span>
<a class="headerlink" href="#typo3-cms-core-test-setdate" title="Permalink to this definition">¶</a>    </dt>
    <dd>

    <p>Set the date.</p>

        </dd>
</dl><dl class="php method">
    <dt class="sig sig-object php" id="typo3-cms-core-test-getdate">
<span class="sig-name descname"><span class="pre">getDate</span></span>
<span class="sig-paren">(</span>
<span class="sig-paren">)</span>
<em class="sig-returns"><span class="pre">: int</span></em>
<a class="headerlink" href="#typo3-cms-core-test-getdate" title="Permalink to this definition">¶</a>    </dt>
    <dd>

    <p>Get the date.</p>

        </dd>
</dl>
    </dd>
</dl>
    </div>
<!-- content end -->
